AdministratorAdministration.yaml

```diff
@@ -245,13 +265,14 @@ paths:
                         - subscriber-application
                         - subscriber-release-number
                         - subscriber-operation
+                        - subscriber-protocol
                         - subscriber-address
                         - subscriber-port
                       properties:
                         subscriber-application:
                           type: string
                           description: >
-                            'Own application name
+                            'Application Name of NewRelease
```

Comment:
PromptForBequeathingDataCausesRObeingRequestedToNotifyApprovalsOfNewApplicationsToNewRelease :: subscriber-application :: description :: 
Path shall be updated to be pointing to NewRelease/ http-client /application-name.

I see it is changed for other callbacks, it should be missed somehow

Reply:
Thanks for indicating.
Changed as proposed.